src/settings.cpp

```diff
@@ -339,7 +368,12 @@ void Settings::DeleteConfigValue(const char *name)
 {
     CriticalSectionLocker lock(g_csConfigValues);
 
-    RegistryDelete(name);
+    if (0 == ms_currentConfigDelete(ms_customConfigData, name))
```

Comment:
Please use either boolean comparison (`if (!…)`) or with the constant at the end (`… == 0)`), as a human would say it (other places too).